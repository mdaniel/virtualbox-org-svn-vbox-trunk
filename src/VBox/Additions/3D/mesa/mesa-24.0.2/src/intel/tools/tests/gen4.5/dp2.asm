dp2(8)          g7<1>.xF        g7<4>.xyyyF     g7<4>.xyyyF     { align16 };
dp2(8)          m5<1>.xF        g1<0>.yF        g1<0>.yF        { align16 };
dp2(8)          m5<1>.yzF       g1<0>.xF        g1<0>.zwwwF     { align16 NoDDClr };
dp2(8)          m5<1>.wF        g1<0>.ywwwF     g1<0>.wyyyF     { align16 NoDDChk };
dp2(8)          g4<1>.yF        g1<0>.xyyyF     g1.4<0>.xyyyF   { align16 NoDDClr };
dp2(8)          g4<1>.zF        g1<0>.xyyyF     g1.4<0>.zwwwF   { align16 NoDDClr,NoDDChk };
dp2(8)          g4<1>.wF        g1<0>.xyyyF     g2<0>.xyyyF     { align16 NoDDChk };
