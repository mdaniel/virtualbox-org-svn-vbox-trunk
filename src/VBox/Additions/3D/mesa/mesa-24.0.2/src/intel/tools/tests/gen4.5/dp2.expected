57 01 60 00 bd 77 e1 20 e4 00 65 00 e4 00 65 00
57 01 60 00 be 77 a1 20 25 00 05 00 25 00 05 00
57 05 60 00 be 77 a6 20 20 00 00 00 2e 00 0f 00
57 09 60 00 be 77 a8 20 2d 00 0f 00 27 00 05 00
57 05 60 00 bd 77 82 20 24 00 05 00 34 00 05 00
57 0d 60 00 bd 77 84 20 24 00 05 00 3e 00 0f 00
57 09 60 00 bd 77 88 20 24 00 05 00 44 00 05 00
